src/index.css

```diff
@@ -27,6 +27,12 @@
   }
 }
 
+@layer utilities {
```

Comment:
@CrazyOptimist when mixing several tailwind classes into a new one the better approach I believe is to create a component like here [Extracting Components - Tailwind CSS](https://tailwindcss.com/docs/extracting-components#extracting-component-classes-with-apply) instead of an utility, which would be a new class with normal CSS to be inserted into the default tailwind utilities classes